internal/controllers/core/cmd/wire.go

```diff
@@ -0,0 +1,7 @@
+package cmd
+
+import "github.com/google/wire"
+
+var WireSet = wire.NewSet(
```

Comment:
I don't think this actually being used currently, but I do like this pattern since make the top level wire cleaner:
```go
var controllerSet = wire.NewSet(
    filewatch.WireSet,
    cmd.WireSet,
    ...
)
```
(I'm assuming controllers will accumulate various sub-dependencies beyond their own ctor)

Reply:
oops, fixed. yep, there will be other deps